bokeh/server/session.py

```diff
@@ -214,7 +213,7 @@ def _handle_patch(self, message, connection):
         self._current_patch = message
         self._current_patch_connection = connection
         try:
-            message.apply_to_document(self.document)
+            message.apply_to_document(self.document, self.id)
```

Comment:
Switched to using ``setter`` rather than ``setter_id`` and checking for identity at least on the python end.